Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2548,6 +2549,7 @@
         - SmallMobLayer
   - type: MobState
   - type: Deathgasp
+  - type: Paper
```

Comment:
wait.. that means that i can put hamlet on a notice board?:trollface:

Reply:
doesnt notice board use tag: document?

Reply:
I know paper bins do

Reply:
> doesnt notice board use tag: document?

ohh, yes :disappointed: